packages/combo-box/test/interactions.test.js

```diff
@@ -0,0 +1,267 @@
+      expect(comboBox.hasAttribute('focused')).to.be.true;
+    });
+
+    it('should blur the input with the blur method', () => {
+      comboBox.focus();
+      comboBox.blur();
+
+      expect(comboBox.hasAttribute('focused')).to.be.false;
+      expect(document.activeElement).to.not.equal(input);
+    });
+
+    it('should focus the input on required indicator click', () => {
+      comboBox.required = true;
+      comboBox.shadowRoot.querySelector('[part="required-indicator"]').click();
+
+      expect(comboBox.hasAttribute('focused')).to.be.true;
+      expect(document.activeElement).to.equal(input);
+    });
+
+    it('should focus the input on opening if not focused', () => {
+      comboBox.open();
+      expect(document.activeElement).to.equal(input);
+    });
+
+    it('should not blur the input on overlay touchstart', () => {
+      comboBox.focus();
+      comboBox.open();
+
+      const spy = sinon.spy(input, 'blur');
+      overlay.dispatchEvent(new CustomEvent('touchstart'));
+      expect(spy.callCount).to.eql(0);
+    });
+
+    it('should blur the input on overlay touchend', () => {
+      comboBox.focus();
+      comboBox.open();
+
+      const spy = sinon.spy(input, 'blur');
+      overlay.dispatchEvent(new CustomEvent('touchend'));
+      expect(spy.callCount).to.eql(1);
+    });
+
+    it('should blur the input on overlay touchmove', () => {
+      comboBox.focus();
+      comboBox.open();
+
+      const spy = sinon.spy(input, 'blur');
+      overlay.dispatchEvent(new CustomEvent('touchmove'));
+      expect(spy.callCount).to.eql(1);
+    });
+
+    it('should prevent default on overlay mousedown', () => {
+      const event = fire(overlay, 'mousedown');
+      expect(event.defaultPrevented).to.be.true;
+    });
+
+    it('should restore focus to the input on outside click', async () => {
+      comboBox.focus();
+      comboBox.open();
+      outsideClick();
+      await aTimeout(0);
+      expect(document.activeElement).to.equal(input);
+    });
+
+    it('should restore focus to the input on toggle button click', async () => {
+      comboBox.focus();
+      comboBox.open();
+      comboBox._toggleElement.click();
+      await aTimeout(0);
+      expect(document.activeElement).to.equal(input);
+    });
+
+    it('should focus the input on outside click if not focused before opening', async () => {
+      expect(document.activeElement).to.equal(document.body);
+      comboBox.open();
+      outsideClick();
+      await aTimeout(0);
+      expect(document.activeElement).to.equal(input);
+    });
+
+    (isIOS ? it : it.skip)('should not focus input on dropdown open if not focused', () => {
+      comboBox.open();
+      expect(document.activeElement).to.not.equal(input);
+    });
+
+    (isIOS ? it : it.skip)('should not restore focus to the input on toggle button click', () => {
+      comboBox.open();
+      tap(comboBox._toggleElement);
+      expect(document.activeElement).to.not.equal(input);
+    });
```

Comment:
These two tests are the only ones that needed `isIOS` check. They are moved from the suite that had this check set incorrectly in #2496 (which was an oversight). Other tests from there are now unskipped.

https://github.com/vaadin/web-components/blob/9a4b06ab8685bae779290b80c7693c3c01b5a439/packages/combo-box/test/toggling-dropdown.test.js#L192